components/scream/src/dynamics/homme/atmosphere_dynamics.cpp

```diff
@@ -142,10 +142,11 @@ void HommeDynamics::set_grids (const std::shared_ptr<const GridsManager> grids_m
   // Note: qv is needed to transform T<->Theta
   add_field<Updated> ("horiz_winds",   FL({COL,CMP, LEV},{ncols,2,nlev_mid}),m/s,   rgn,N);
   add_field<Updated> ("T_mid",         FL({COL,     LEV},{ncols,  nlev_mid}),K,     rgn,N);
-  add_field<Updated> ("phi_int",       FL({COL,    ILEV},{ncols,  nlev_int}),Pa/rho,rgn,N);
   add_field<Updated> ("pseudo_density",FL({COL,     LEV},{ncols,  nlev_mid}),Pa,    rgn,N);
   add_field<Updated> ("ps",            FL({COL         },{ncols           }),Pa,    rgn);
   add_field<Required>("qv",            FL({COL,     LEV},{ncols,  nlev_mid}),Q,     rgn,"tracers",N);
+  add_field<Required>("phis",          FL({COL         },{ncols           }),Pa/rho,rgn);
+  add_field<Computed>("phi_int",       FL({COL,    ILEV},{ncols,  nlev_int}),Pa/rho,rgn,N);
```

Comment:
So is the `phi_int` field still needed as even a computed field? I don't think any other AD process or SC uses it.

Reply:
Right, it's not really needed. Just like `w_int`, tbf. I have wondered whether I should purge them or leave them. In terms of memory, it's "only" 2 phys grid fields. It adds 2 fields to remap during PD remap though. Doesn't seem like a huge waste to me. But I guess waste can add up over time.

@AaronDonahue what are your thoughts? Should we remove `phi_int` and `w_int` on the phys grid?

Reply:
We will probably want to write out w_int and maybe phi_int for analysis, but we can just do that on the dynamics grid anyway, right? And better to output stuff as close to its native environment as possible....

Reply:
I'm with @PeterCaldwell .  I don't think we should get in the habit of remapping variables to physics that are not actually needed in physics.  If the need arises we can make that happen then.  But EAM is pretty rife with clutter of fields added to PBUF that are not used and fields added as possible output that may be outdated.  So as Luca pointed out, it's only 2 now but those can quickly add up and add more confusion than anything else.

Reply:
@PeterCaldwell good call, I forgot about that feature (despite being the one that added it!). Then I will remove the phys grid version of phi_int and w_int from HommeDynamics. If one is just interested in outputing them, they can request to do so from file (even if they want it outputed on the phys grid!).